reader-gtfs/src/main/java/com/graphhopper/reader/gtfs/GraphHopperGtfs.java

```diff
@@ -341,6 +339,10 @@ public static GraphHopperStorage createOrLoad(GHDirectory directory, EncodingMan
             for (int i = 0; i < id; i++) {
                 new GtfsReader("gtfs_" + i, graphHopperStorage, gtfsStorage, ptFlagEncoder, walkNetworkIndex).readGraph();
             }
+            // This currently needs to happen as the last step, since we cannot add new nodes after this step.
+            // This means that disconnected parts of the transit network are removed as well.
+            // If we don't want that, we need to think of something.
+            new PrepareRoutingSubnetworks(graphHopperStorage, Collections.singletonList(ptFlagEncoder)).doWork();
```

Comment:
Strange, why does it revert your change @michaz although I merged master? 

https://github.com/graphhopper/graphhopper/commit/506489399629519804b44ff1cf5058b79db46cf5